kogito-editors-js/pom.xml

```diff
@@ -21,9 +21,7 @@
     <sonar.test.inclusions>**/*test.ts,**/*test.tsx</sonar.test.inclusions>
     <sonar.javascript.lcov.reportPaths>
       packages/boxed-expression-component/coverage/lcov.info,
-      packages/dmn-loader/coverage/lcov.info,
       packages/feel-input-component/coverage/lcov.info,
-      packages/hello-world-component/coverage/lcov.info,
```

Comment:
@jomarko Can you please clarify these 2 removals?

Reply:
I am not really up to date on this guys, so I agree with Yeser about iinfo that look a bit weird. Anyway, adding coverage looks good to me, if this PR does not actually prevents from building acutal repos. :+1:

Reply:
well, my intention was somehow optimize/minimize execution time. I know from global point of view (wating for whole CI) it is maybe even not noticeable, but I think we should try  to improve CI in such small steps as here, here are my assumptions, but I am open to your comments and suggestions:
- **dmn-loader** has no tests, so from my point of view doesn't make sense to try measure the coverage there
- **hello-world-component** has no tests and furthermore it is just a showcase not present in our product at all